com.unity.netcode.gameobjects/Prototyping/NetworkTransform.cs

```diff
@@ -21,106 +24,159 @@ internal struct NetworkState : INetworkSerializable
+                set => Bitset |= (ushort) ((value ? 1 : 0) << InLocalSpaceBit);
             }
+
             // Position
             public bool HasPositionX
             {
                 get => (Bitset & (1 << PositionXBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << PositionXBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << PositionXBit);
             }
+
             public bool HasPositionY
             {
                 get => (Bitset & (1 << PositionYBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << PositionYBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << PositionYBit);
             }
+
             public bool HasPositionZ
             {
                 get => (Bitset & (1 << PositionZBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << PositionZBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << PositionZBit);
             }
+
             // RotAngles
             public bool HasRotAngleX
             {
                 get => (Bitset & (1 << RotAngleXBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << RotAngleXBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << RotAngleXBit);
             }
+
             public bool HasRotAngleY
             {
                 get => (Bitset & (1 << RotAngleYBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << RotAngleYBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << RotAngleYBit);
             }
+
             public bool HasRotAngleZ
             {
                 get => (Bitset & (1 << RotAngleZBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << RotAngleZBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << RotAngleZBit);
             }
+
             // Scale
             public bool HasScaleX
             {
                 get => (Bitset & (1 << ScaleXBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << ScaleXBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << ScaleXBit);
             }
+
             public bool HasScaleY
             {
                 get => (Bitset & (1 << ScaleYBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << ScaleYBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << ScaleYBit);
             }
+
             public bool HasScaleZ
             {
                 get => (Bitset & (1 << ScaleZBit)) != 0;
-                set => Bitset |= (ushort)((value ? 1 : 0) << ScaleZBit);
+                set => Bitset |= (ushort) ((value ? 1 : 0) << ScaleZBit);
             }
 
             public float PositionX, PositionY, PositionZ;
             public float RotAngleX, RotAngleY, RotAngleZ;
             public float ScaleX, ScaleY, ScaleZ;
+            public double SentTime;
+
+            public Vector3 Position
```

Comment:
Are we sure we want the `Position`, `Rotation` and `Scale` properties public? What would they be used for compared from just getting these values from the transform?

Reply:
nah, these values are stored on the `NetworkTransform.NetworkState`, not on the actual `transform` itself. I actually like these new utility properties, they make sense.